string fileName
string path
uint32 timeToRecord
uint32 xSize
uint32 ySize